docs/fundamentals/write-operations.txt

```diff
@@ -480,3 +479,134 @@ with ``"contemporary"`` in the ``genres`` array field. Click the
 
 To learn more about array update operators, see :manual:`Array Update Operators </reference/operator/update-array/>`
 in the {+server-docs-name+}.
+
+.. _laravel-fundamentals-delete-documents:
+
+Delete Documents
+----------------
+
+In this section, you can learn how to delete documents from a MongoDB collection
+by using {+odm-short+}. Use delete operations to remove data from your MongoDB
+database permanently.
+
+This section provides examples of the following delete operations:
+
+- :ref:`Delete one document <laravel-fundamentals-delete-one>`
+- :ref:`Delete multiple documents <laravel-fundamentals-delete-many>`
+
+To learn about the Laravel features available in {+odm-short+} that modify
+delete behavior, see the following sections:
+
+- :ref:`Soft delete <laravel-model-soft-delete>`, which lets you mark
+  documents as deleted instead of removing them from the database
+- :ref:`Pruning <laravel-model-pruning>`, which lets you define conditions
+  that qualify a document for automatic deletion
+
+.. _laravel-fundamentals-delete-one:
+
+Delete a Document Examples
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You can delete one document in the following ways:
+
+- Call the ``destroy()`` method on the model, passing it the primary key of
+  the document to be deleted.
+- Call the ``delete()`` method on an instance of the model.
+- Chain methods to retrieve and delete an instance of a model by calling the
+  ``delete()`` method.
```

Comment:
`$model->delete()` should be the first in the list as it's probably the more used.

The alternate `Model::destroy($ids)` accepts a single id, a list of ids, or a list of model intances.

```suggestion
- Call the ``$model->delete()`` method on an instance of the model.
- Call the ``Model::destroy($ids)`` method on the model class, passing it the id of
  the document to be deleted.
- Chain methods to retrieve and delete an instance of a model by calling the
  ``delete()`` method.
```

Reply:
Question:
Is the "$model->delete()" a common way to represent a placeholder for an Eloquent model for a Laravel developer audience? Otherwise, what would be a common way to explain calling on a method on an instance of a model?

Reply:
I believe that code is more explicit than words. This is the convention I use systematically on this project when I want to emphasize the scope of a method.

I've found examples of `Model::method` in [Eloquent doc](https://laravel.com/docs/11.x/eloquent), but not `$model->method`. See https://laravel.com/docs/11.x/eloquent#deleting-models

Reply:
I'm fine with changing conventions, but is this something we could come back to and decide later? I think it might involve updating several of the currently published docs to make them use it consistently. If so, I can ticket that.